### DISCUSSION TITLE
What is the best way to work with form fields within a custom node?

### DISCUSSION THREAD

**Discussion by dkrusenstrahle** [ueberdosis/tiptap#1256] [General Questions]
Hello,

I have created a custom view node that got a form with one field within it. This field will take a link and extract meta data (via API request) and then display it to the user. Unfortunately Tiptap 2.0 keeps hijacking the focus so when for example the user pastes a link into the field it is pasted into the paragraph above it and when I type into the field back space (delete characters) does not work.

What is the best way to work with form fields within a custom node?

**Comment by philippkuehn** [comment]
Hmmm, tiptap should allow pasting into input elements:

https://github.com/ueberdosis/tiptap/blob/bb116ec471e9ea76d2e0039672d57a71c9c22ae9/packages/core/src/NodeView.ts#L116-L122

Can you provide a codesandbox?

**Reply by dkrusenstrahle** [reply]
So I guess I am not adding my viewnode correctly into the editor.
This is my Node.create file: https://codeshare.io/axDXlB

**Reply by philippkuehn** [reply]
It seems that there was a broken version which will lead to your issue: https://github.com/sibiraj-s/ngx-tiptap/commit/d385e105fc76e83b028dfab217daae9d8f4fc5ef#diff-c63cc7cf40328cb095aaad64801b1a033f4f06803b592167fc26575134f001fa

**Reply by dkrusenstrahle** [reply]
@philippkuehn That update seems to have fixed the issue! Thank you for making me see it.
